pytorch_lightning/plugins/plugin_connector.py

```diff
@@ -31,8 +31,8 @@ def __init__(self, trainer):
         self.plugins = []
         self.ddp_plugin = DDPPlugin()
         self.cloud_environment = None
-        self.amp_plugin = NativeAMPPlugin(trainer)
-        self.apex_plugin = ApexPlugin(trainer)
+        # self.amp_plugin = NativeAMPPlugin(trainer)
```

Comment:
should these be removed?

Reply:
I think yes because it is not used anywhere in codebase

Reply:
https://github.com/PyTorchLightning/pytorch-lightning/pull/5297#discussion_r551408297